packages/plugin-analytic-fb-pixel/README.md

```diff
@@ -0,0 +1,104 @@
+# ima-plugin-analytic-fb-pixel
+
+This is the Facebook Pixel analytic plugin for the IMA.js application. You can find the IMA.js skeleton application at <https://github.com/seznam/IMA.js-skeleton>
+or follow link <https://imajs.io>.
+
+## Installation
+
+```javascript
+
+npm install ima-plugin-analytic-fb-pixel ima-plugin-script-loader --save
+
+```
+
+```javascript
+// /app/build.js
+
+var vendors = {
+	common: [
+		'ima-plugin-analytic-fb-pixel',
+		'ima-plugin-analytic',
+		'ima-plugin-script-loader'
+	]
+};
+
+/*
+Now is FB Pixel analytic plugin available from:
+
+import { FacebookPixelAnalytic, defaultDependencies } from 'ima-plugin-analytic-fb-pixel';
+*/
+
+```
+
+```javascript
+// /app/config/settings.js
+
+prod: {
+	$Http: { ... },
+	$Cache: { ... },
+	$Page:{ ... },
+	plugin : {
+		analytic: {
+			fbPixel: {
+				id: 'XXX'
+			}
+		}
+	}
+}
+```
+
+```javascript
```

Comment:
Why must you create alias?

Reply:
1. Following same pattern as in `ima-plugin-analytic-google`
2. `bind` is the only method that doesn't override default dependencies.